vesin/CMakeLists.txt

```diff
@@ -80,6 +99,25 @@ target_include_directories(vesin_objects PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
 
+if (CMAKE_CUDA_COMPILER AND VESIN_ENABLE_CUDA)
+    # Include gpuLite headers for both targets
+    target_include_directories(vesin PRIVATE external/gpuLite)
```

Comment:
instead of vendoring the code, I would rather use `fetch_content`. Do you know how to do this, or should I do give it a go?